lightningd/test/run-jsonrpc.c

```diff
@@ -150,9 +151,10 @@ static int test_json_filter(void)
 	str = json_out_contents(result->jout, &len);
 	str = tal_strndup(result, str, len);
 
-	toks = json_parse_input(str, str, strlen(str), &valid);
+	toks = toks_alloc(str);
+	jsmn_init(&parser);
+	valid = json_parse_input(&parser, &toks, str, strlen(str), NULL);
```

Comment:
NULL will be dereferenced, maybe `json_parse_input_simple` ?

Reply:
Same below

Reply:
(did not submit the review)...

Reply:
YEah, originally instead of the json_parse_simple() API I allowed NULL args for parser and complete.  But I reverted that, and of course, missed this :(

Fixed in the obvious way now...